packages/core/saved-objects/core-saved-objects-migration-server-internal/src/document_migrator/build_active_migrations.test.ts

```diff
@@ -0,0 +1,322 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import {
+  getConversionTransformsMock,
+  getModelVersionTransformsMock,
+  getReferenceTransformsMock,
+  resetAllMocks,
+  validateTypeMigrationsMock,
+} from './build_active_migrations.test.mocks';
+
+import { loggerMock, MockedLogger } from '@kbn/logging-mocks';
+import type { SavedObjectsType } from '@kbn/core-saved-objects-server';
+import { buildActiveMigrations } from './build_active_migrations';
+import { SavedObjectTypeRegistry } from '@kbn/core-saved-objects-base-server-internal';
+import { Transform, TransformType } from './types';
+
+const kibanaVersion = '3.2.3';
+
+describe('buildActiveMigrations', () => {
+  let log: MockedLogger;
+  let typeRegistry: SavedObjectTypeRegistry;
+
+  const buildMigrations = () => {
+    return buildActiveMigrations({ typeRegistry, kibanaVersion, log });
+  };
+
+  const createType = (parts: Partial<SavedObjectsType>): SavedObjectsType => ({
```

Comment:
nit: these helpers are redeclared in `model_version.test.ts` and could be extracted to a "test_helpers" file.